metaseq/models/transformer_decoder.py

```diff
@@ -192,12 +192,39 @@ def __init__(self, args, dictionary, embed_tokens):
             if initialize_params_on_gpu:
                 alibi = alibi.cuda()
             self.alibi = torch.nn.Parameter(alibi)
```

Comment:
I probably should've used an Embedding layer in order to get better performance.